addons/web/static/tests/views/form_tests.js

```diff
@@ -4854,6 +4854,70 @@ QUnit.module('Views', {
         form.destroy();
     });
 
+    QUnit.test('correct amount of buttons', function (assert) {
+        assert.expect(7);
+
+        var buttons = '';
+
+        for(var i = 0; i < 7; i++) {
+            buttons += '<button type="object" class="oe_stat_button" icon="fa-check-square">' +
+                '<field name="bar"/>' +
+            '</button>';
+        }
+
+        var createFormWithDeviceSizeClass = function (self, size_class) {
```

Comment:
......
it is not necessary to pass the self argument here anymore

Reply:
@kebeclibre Changes done! I cleaned up a little bit too